app/models/like.rb

```diff
@@ -0,0 +1,14 @@
+class Like < ApplicationRecord
+  belongs_to :post, counter_cache: true
+  belongs_to :user
+
+  def update_likes_counter
+    post.update(likes_counter: post.likes_counter + 1)
+  end
```

Comment:
- [ ] Just like you did in the Post model, please use `after_save :update_likes_counter`